server/CGameHandler.cpp

```diff
@@ -5997,6 +5992,18 @@ void CGameHandler::putArtifact(const ArtifactLocation &al, const CArtifactInstan
 	sendAndApply(&pa);
 }
 
+bool CGameHandler::giveHeroNewArtifact(const CGHeroInstance* h, const CArtifact* art)
```

Comment:
Since we agree on formatting of pointers that would be good to keep them consistent in new code.